main.go

```diff
@@ -66,7 +68,9 @@ func run() error {
 	pflag.StringVar(&opt.ForcePROwner, "force-pr-owner", opt.ForcePROwner, "Make the supplied user the owner of all PRs for access control purposes.")
 	pflag.StringVar(&ignored, "build-cluster-kubeconfig", ignored, "REMOVED: Kubeconfig to use for buildcluster. Defaults to normal kubeconfig if unset.")
 	pflag.StringVar(&opt.BuildClusterKubeconfigsLocation, "build-cluster-kubeconfigs-location", opt.BuildClusterKubeconfigsLocation, "Path to the location of the Kubeconfigs for the various buildclusters. Default is \"/var/build-cluster-kubeconfigs\".")
-	pflag.StringVar(&opt.ReleaseClusterKubeconfig, "release-cluster-kubeconfig", "", "Kubeconfig to use for cluster housing the release imagestreams. Defaults to normal kubeconfig if unset.")
+	pflag.StringVar(&opt.ReleaseClusterKubeconfig, "release-cluster-kubeconfig", opt.ReleaseClusterKubeconfig, "Kubeconfig to use for cluster housing the release imagestreams. Defaults to normal kubeconfig if unset.")
+	pflag.StringVar(&opt.Command, "command", opt.Command, "The command to run in command-line mode")
+	pflag.StringVar(&opt.Output, "output", opt.Output, "The path, of the file, where to write the prow job JSON")
```

Comment:
Why aren't you just outputting this to stdout?  Is there extra info you don't need?

Reply:
If the `--output` option is not specified, then I was writing the resultant prowjob to the screen.  I updated the logic to write directly to stdout instead of using klog.

Reply:
The comment was because we probably don't need the `--output` flag at all.  Just output prow jobs to stdout when --command is provided.